config/locales/en.yml

```diff
@@ -1040,109 +1040,109 @@ en:
           retail: "Retail Area"
           village_green: "Village Green"
           vineyard: "Vineyard"
-          "yes": "Landuse"
+          "yes": "Land Use Area"
         leisure:
-          adult_gaming_centre: "Adult Gaming Centre"
+          adult_gaming_centre: "Adult Gaming Center"
           amusement_arcade: "Amusement Arcade"
           bandstand: "Bandstand"
           beach_resort: "Beach Resort"
           bird_hide: "Bird Hide"
           bleachers: "Bleachers"
           bowling_alley: "Bowling Alley"
-          common: "Common Land"
+          common: "Common"
           dance: "Dance Hall"
           dog_park: "Dog Park"
           firepit: "Fire Pit"
-          fishing: "Fishing Area"
-          fitness_centre: "Fitness Centre"
-          fitness_station: "Fitness Station"
+          fishing: "Fishing Hole"
+          fitness_centre: "Fitness Center"
+          fitness_station: "Outdoor Fitness Station"
           garden: "Garden"
           golf_course: "Golf Course"
-          horse_riding: "Horse Riding Centre"
+          horse_riding: "Horseback Riding Center"
           ice_rink: "Ice Rink"
           marina: "Marina"
           miniature_golf: "Miniature Golf"
-          nature_reserve: "Nature Reserve"
-          outdoor_seating: "Outdoor Seating"
+          nature_reserve: "Nature Preserve"
+          outdoor_seating: "Outdoor Seating Area"
           park: "Park"
           picnic_table: "Picnic Table"
-          pitch: "Sports Pitch"
+          pitch: "Playing Field"
```

Comment:
I'm not sure if this works.  Though `leisure=pitch` is used for many "sports fields" or "playing fields", it is also used for tennis courts, basketball courts, hockey rinks, and perhaps other sports areas that aren't fields.  It seems that in  British English the term "pitch" covers all these area types but we don't have an equivalent umbrella term in US English.

Reply:
In standard British English, can “pitch” actually refer to a badminton court or horseback riding arena? I thought that was “mapping speak” anyways. We can make it “Sports Field or Court” if that would be more understandable. I was trying to avoid equivocation in these strings, but this is a relatively good case for conjoining two terms.

Reply:
The [Wikipedia page on pitches](https://en.wikipedia.org/wiki/Pitch_(sports_field)) includes badminton and various other court sports.  Of course the article "needs additional citations" so who knows.  The [Cambridge ](https://dictionary.cambridge.org/dictionary/english/pitch)& [Collins ](https://www.collinsdictionary.com/dictionary/english/pitch)dictionaries both define "pitch" as an area with painted lines or markings for playing a sport.  This sounds to me like a court would fit this description since a specific surface type is not stated.  They both include hockey (played on an ice rink not a field) as an example.  I'm not sure exactly what the term "pitch" covers in British English, but it does seem to have broader meaning than "playing field" which typically indicates a grass or turf surface.  I don't think I've ever heard a basketball or tennis court referred to as a playing field, but maybe it is technically still correct when discussing the rules of the sport and the "field of play".

Reply:
"Sports Pitch" is a confusing term to me as an American English speaker, even if there are nuanced differences I think `Playing Field` is clearer overall.